tests/test_model.py

```diff
@@ -370,6 +370,40 @@ def test_setup_region(model, demda, tmpdir):
     assert np.all(model.region["value"] == 210000039)  # basin id
 
 
+def test_model_write_geoms(tmpdir):
+    model = Model(root=str(tmpdir), mode="w")
+    bbox = box(*[4.221067, 51.949474, 4.471006, 52.073727])
+    geom = gpd.GeoDataFrame(geometry=[bbox], crs=4326)
+    geom.to_crs(epsg=28992, inplace=True)
+    fn_region = str(join(tmpdir, "region.gpkg"))
+    geom.to_file(fn_region, driver="GPKG")
+    model.data_catalog.from_dict(
+        {
+            "region": {
+                "path": fn_region,
+                "data_type": "GeoDataFrame",
+                "driver": "vector",
+            }
+        }
+    )
+    model.setup_region({"geom": "region"})
```

Comment:
this should also work:
```suggestion
    model.setup_region({"geom": geom})
```

or even more simplified:
```py    
    model.set_geoms({"region": geom})
```

Reply:
I changed it to `model.set_geoms(geom=geom, name="region")`